external-adapter/src/requester.ts

```diff
@@ -53,7 +53,11 @@ export class Requester {
       }
 
       // Success
-      logger.info(`Received response: ${JSON.stringify(response.data)}`)
+      const { request: r, ...rest } = response
+      logger.debug({
+        message: 'Received response',
+        ...rest,
```

Comment:
We logged just the `response.data` before. Why the change?

Reply:
This is also short enough to be a one-liner.

Reply:
In my testing when the data response was logged it includes the `request` object which is a massive (200+ lines) reference to Axios that doesn't seem add to present any useful information. This takes out the request property.

Reply:
Ok, that makes a lot of sense.

This is what the [AxiosResponse](https://github.com/axios/axios/blob/fe52a611efe756328a93709bbf5265756275d70d/index.d.ts#L76) looks like. I would suggest to log only the `{ data, status, statusCode }`, and remove the `request`, `config` and `headers` as those could leak secrets out.